scripts/devaccess/helper_scripts/redact_env_vars.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+This script performs inline filtering of the yaml output of an openshift resource
+and redacts the values of env vars likely to contain unfiltered passwords
+"""
+import sys
+import yaml
+
```

Comment:
Add the following lines so that pylint stops preventing the PR validation. The code looks fine.

```suggestion
#pylint: disable=invalid-name,
#pylint: disable=superfluous-parens
```